src/createRelease.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const axios = require('axios');
+const config = require('../.config/.config.js');
+const { PAT } = config;
+
+
+async function createRelease(changelog) {
+    const array = changelog.split('\n');
```

Comment:
The changelog parsing logic shouldn't be in `createRelease()`. Move that out to `createReleaseFromChangelog()`. This function should just be responsible for making the HTTP request, and take the `tagAndName` and `body` as parameters